### ISSUE TITLE
Error retrieving card details

### ISSUE THREAD

**Issue by JJJGGGG** [aleho/gnome-shell-volume-mixer#131]
Hello. I installed this extension as shown, then copied the folder to ```~/.local/share/gnome-shell/extensions/```. I then opened gnome-tweaks and tried to enable the extension, but an error message showed involving pulseaudio (I didn't catch which error nor I am able to reproduce it). Next I tried opening configurations->devices and i am always presented with an error that says "error retrieving card details, helper script did not return valid card data".

More info: 
OS: POP!_OS 20.04
inxi -Fxz
```sh
System:
  Kernel: 5.8.0-7630-generic x86_64 bits: 64 compiler: N/A 
  Desktop: Gnome 3.36.4 Distro: Pop!_OS 20.04 LTS 
  base: Ubuntu 20.04 LTS Focal 
Machine:
  Type: Laptop System: HP product: HP Laptop 14-cf0xxx 
  v: Type1ProductConfigId serial: <filter> 
  Mobo: HP model: 84B3 v: KBC Version 74.30 serial: <filter> UEFI: Insyde 
  v: F.42 date: 06/20/2019 
Battery:
  ID-1: BAT0 charge: 28.5 Wh condition: 36.7/36.7 Wh (100%) 
  model: HP Primary status: Discharging 
CPU:
  Topology: Dual Core model: Intel Core i3-7020U bits: 64 type: MT MCP 
  arch: Kaby Lake rev: A L2 cache: 3072 KiB 
  flags: avx avx2 lm nx pae sse sse2 sse3 sse4_1 sse4_2 ssse3 vmx 
  bogomips: 18399 
  Speed: 800 MHz min/max: 400/2300 MHz Core speeds (MHz): 1: 799 2: 800 
  3: 798 4: 799 
Graphics:
  Device-1: Intel HD Graphics 620 vendor: Hewlett-Packard driver: i915 
  v: kernel bus ID: 00:02.0 
  Display: x11 server: X.Org 1.20.8 driver: i915 resolution: 1366x768~60Hz 
  OpenGL: renderer: Mesa Intel HD Graphics 620 (KBL GT2) 
  v: 4.6 Mesa 20.2.3 - kisak-mesa PPA direct render: Yes 
Audio:
  Device-1: Intel Sunrise Point-LP HD Audio vendor: Hewlett-Packard 
  driver: snd_hda_intel v: kernel bus ID: 00:1f.3 
  Sound Server: ALSA v: k5.8.0-7630-generic 
Network:
  Device-1: Realtek RTL8111/8168/8411 PCI Express Gigabit Ethernet 
  vendor: Hewlett-Packard driver: r8169 v: kernel port: 4000 bus ID: 02:00.0 
  IF: enp2s0 state: down mac: <filter> 
  Device-2: Realtek RTL8723DE 802.11b/g/n PCIe Adapter 
  vendor: Hewlett-Packard driver: rtw_8723de v: N/A port: 3000 
  bus ID: 03:00.0 
  IF: wlp3s0 state: up mac: <filter> 
  Device-3: Realtek 802.11n WLAN Adapter type: USB driver: btusb 
  bus ID: 1-8:3 
Drives:
  Local Storage: total: 931.51 GiB used: 66.07 GiB (7.1%) 
  ID-1: /dev/sda vendor: Seagate model: ST1000LM035-1RK172 size: 931.51 GiB 
  temp: 37 C 
RAID:
  Hardware-1: Intel 82801 Mobile SATA Controller [RAID mode] driver: ahci 
  v: 3.0 bus ID: 00:17.0 
Partition:
  ID-1: / size: 718.54 GiB used: 65.82 GiB (9.2%) fs: ext4 dev: /dev/sda3 
Sensors:
  System Temperatures: cpu: 38.5 C mobo: N/A gpu: amdgpu temp: 45 C 
  Fan Speeds (RPM): N/A 
Info:
  Processes: 313 Uptime: 1d 1h 26m Memory: 3.72 GiB used: 2.71 GiB (72.9%) 
  Init: systemd runlevel: 5 Compilers: gcc: 9.3.0 Shell: bash v: 5.0.17 
  inxi: 3.0.38 
```

**Comment by aleho** [comment]
Did you install from master?

**Comment by JJJGGGG** [comment]
Yes, I installed from main.

**Comment by aleho** [comment]
There was a bug in master. Pull the latest version please and check again. At least the settings should work again.

If it still doesn't activate I'll  be needing at least some gnome-shell logs to analyze it further.

**Comment by jdoe0000000** [comment]
I had a similar (possibly the same) issue. In my case, when I tried to switch outputs, I get an error that says `error: Volume Mixer: Querying PulseAudio sound cards failed, disabling extension`.

I did a little bit of investigation, and It looks like this is caused by missing Python scripts in the generated zip file; the current Makefile only includes pautils/pa.py in the zip. Once I copy over all the remaining Python scripts, the error went away.
